test/functional/wallet_sendall.py

```diff
@@ -284,6 +284,12 @@ def sendall_fails_on_high_fee(self):
                 recipients=[self.remainder_target],
                 fee_rate=100000)
 
+    @cleanup
+    def sendall_fails_on_low_fee(self):
+        self.log.info("Test sendall fails if the transaction fee is too low")
```

Comment:
nit: would prefer to be precise
```suggestion
        self.log.info("Test sendall fails if the transaction fee is lower than the minimum fee rate setting")
```

Reply:
Done